src/Raven.Server/ServerWide/ClusterCommandsVersionManager.cs

```diff
@@ -187,7 +187,8 @@ public sealed class ClusterCommandsVersionManager
             [nameof(UpdateResponsibleNodeForTasksCommand)] = UpdateResponsibleNodeForTasksCommand.CommandVersion,
             [nameof(AddPrefixedShardingSettingCommand)] = 62_000,
             [nameof(DeletePrefixedShardingSettingCommand)] = 62_000,
-            [nameof(UpdatePrefixedShardingSettingCommand)] = 62_000
+            [nameof(UpdatePrefixedShardingSettingCommand)] = 62_000,
+            [nameof(RevisionsBinConfigurationCommand)] = 62_000
```

Comment:
we need to bump this to at least `62_001`

Reply:
Done